css/styles.css

```diff
@@ -0,0 +1,162 @@
+body {
+  background: rgb(208, 231, 239);
+  font-family: 'Times New Roman', Times, serif;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+}
+
+.container {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.row2 {
+  width: 100%;
+  text-align: center;
+}
```

Comment:
Ideally you don't have this named `.row2` and it's named something more description. This could also be broken up into two separate utility classes like `text-center` that is `text-align: center` and if this is in a `flex container`, you could do a `column-full` with `flex-basis: 100%` or just have a different name with a `width: 100%`